mappings/net/minecraft/world/gen/feature/StructureSpacingInformation.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_5314 net/minecraft/world/gen/feature/StructureSpacingInformation
```

Comment:
Is the field called `information` in the codec for structure spacing config?

Reply:
It doesn't have a clear name in the Codec as far as I can tell? I think ```Information``` just makes sense as a suffix.